docs/get_started/install.md

```diff
@@ -424,13 +443,214 @@ $ sudo python setup.py install
+
+<div class="virtualenv">
+<br/>
+
+**Step 1**  Install prerequisites - Homebrew, python development tools.
+
+```bash
+# Install Homebrew
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+$ export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+
+# Install python development tools - python2.7, pip, python-setuptools
+$ brew install python
+```
+
+**Step 2**  Install virtualenv for macOS.
+
+```bash
+$ pip install virtualenv
+```
+
+**Step 3**  Create and activate virtualenv environment for MXNet.
+
+Following command creates a virtualenv environment at `~/mxnet` directory. However, you can choose any directory by replacing `~/mxnet` with a directory of your choice.
+
+```bash
+$ virtualenv --system-site-packages ~/mxnet
+```
+
+Activate the virtualenv environment created for *MXNet*.
+
+```bash
+$ source ~/mxnet/bin/activate
+```
+
+After activating the environment, you should see the prompt as below.
+
+```bash
+(mxnet)$
+```
+
+**Step 4**  Install MXNet in the active virtualenv environment.
+
+Installing *MXNet* with pip requires a latest version of `pip`. Install the latest version of `pip` by issuing the following command.
+
+```bash
+(mxnet)$ pip install --upgrade pip
+(mxnet)$ pip install --upgrade setuptools
+```
+
+Install *MXNet* with OpenBLAS acceleration.
+
+```bash
+(mxnet)$ pip install mxnet
+```
+
+**Step 5**  Validate the installation by running simple *MXNet* code described [here](#validate-mxnet-installation).
+
+**Note**  You can read more about virtualenv [here](https://virtualenv.pypa.io/en/stable/userguide/).
 
 </div>
 
-<div class="linux">
+<div class="pip">
+<br/>
+
+**Step 1**  Install prerequisites - Homebrew, python development tools.
+
+```bash
+# Install Homebrew
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+$ export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+
+# Install python development tools - python2.7, pip, python-setuptools
+$ brew install python
+```
+
+**Step 2** Install MXNet with OpenBLAS acceleration.
+
+Installing *MXNet* with pip requires a latest version of `pip`. Install the latest version of `pip` by issuing the following command.
+
+```bash
+$ pip install --upgrade pip
+$ pip install --upgrade setuptools
+```
+
+```bash
+$ pip install mxnet
+```
+
+**Step 3**  Validate the installation by running simple MXNet code described [here](#validate-mxnet-installation).
+
+</div>
+
+<div class="docker">
+<br/>
+
+Docker images with *MXNet* are available at [Docker Hub](https://hub.docker.com/r/mxnet/).
+
+**Step 1**  Install Docker on your machine by following the [docker installation instructions](https://docs.docker.com/docker-for-mac/install/#install-and-run-docker-for-mac).
```

Comment:
What do you think about installing docker with homebrew? That way we have one more requirement that we can give concrete commands for.